src/eslint-plugin-adeira/src/rules/__tests__/graphql-require-description.test.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+
+const RuleTester = require('eslint').RuleTester;
+
+const rule = require('../graphql-require-description');
+
+const ruleTester = new RuleTester({
+  parser: require.resolve('babel-eslint'),
+});
+
+ruleTester.run('graphql-require-description', rule, {
+  valid: [
+    `new Date`,
+    `new Pokus({ foo: { baz: 'bar' }})`,
+    `new GraphQLObjectType('Random stuff')`,
+    `
+    new GraphQLObjectType({
+      name: 'TripSegment',
+      description: '"TripSegment" represents single atomic connection between two places in itinerary.',
+      fields: {
+        departure: {
+          type: GraphQLRouteStop,
+          resolve: ({ departure }: TripData): DepartureArrival => departure,
+        },
+      },
+    });
+  `,
+  ],
+
+  invalid: [
+    {
+      code: `
+        new GraphQLObjectType({
+          name: 'TripSegment',
+          fields: {
+            departure: {
+              type: GraphQLRouteStop,
+              resolve: ({ departure }: TripData): DepartureArrival => departure,
+            },
+          },
+        });
+  `,
+      errors: [
```

Comment:
What if I add `description: ''`

Reply:
you are right, I realized it later, will fix, thanks!

Reply:
should be fixed now!